lib/src/components/NxSmallTag/types.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { LabelHTMLAttributes, ReactNode } from 'react';
+import * as PropTypes from 'prop-types';
+
+export const NX_SMALL_TAG_COLORS = ['blue', 'green', 'indigo', 'orange', 'pink',
+  'purple', 'red', 'teal', 'turquoise'] as const;
```

Comment:
A cleaner way to break the lines
```suggestion
export const NX_SMALL_TAG_COLORS =
  ['blue', 'green', 'indigo', 'orange', 'pink', 'purple', 'red', 'teal', 'turquoise'] as const;
```